### PR TITLE
GH-986 Fix Build (WordCloud Error)

### PR DESCRIPTION
**Pull request by mlzummo**
Fix Build (WordCloud Error)

Fixes GH-986

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/other/WordCloud.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
     } else {
         root.other_WordCloud = factory(root.d3, root.common_SVGWidget, root.other_IWordCloud);
     }
-}(this, function (d3, SVGWidget, IWordCloud) {
+}(this, function (d3, SVGWidget, IWordCloud, d3Cloud) {
```
Comment (GordonSmith) [inline src/other/WordCloud.js]: Just inspecting the code I don't think this is needed?

File: src/other/WordCloud.js [hunk 2]
```diff
@@ -38,7 +38,12 @@
     };
 
     WordCloud.prototype.enter = function (domNode, element) {
-        this.cloud = d3.layout.cloud()
+        if (typeof define === "function" && define.amd) {
+            this.cloud = d3Cloud();
+        } else {
+            this.cloud = d3.layout.cloud();
+        }
+        this.cloud
```
Comment (GordonSmith) [inline src/other/WordCloud.js]: Again just using code inspection I don't think this is needed either.

File: src/other/WordCloud.js (hunk 1 above)
Comment (mlzummo) [inline src/other/WordCloud.js]: if not defined there you will get an error about d3Cloud not being defined below

File: src/other/WordCloud.js (hunk 2 above)
Comment (mlzummo) [inline src/other/WordCloud.js]: d3 changed the way amd/nonamd work

**Comment by mlzummo** [comment]
@GordonSmith Fixed and Ready to be pulled

**Comment by GordonSmith** [comment]
Closing in favor of:  https://github.com/hpcc-systems/Visualization/pull/996

**Pull request closed without merging**